### DISCUSSION TITLE
Get the latest released agent service

### DISCUSSION THREAD

**Discussion by barnabaskugler** [microsoft/azure-pipelines-agent#4389] [General]
Hi, we would like to have the latest released agent services downloaded all the time.
Now I see a discrepancy between Azure DevOps Services and the GitHub API result.

https://api.github.com/repos/Microsoft/azure-pipelines-agent/releases/latest give back version 3.220.5 however from the Azure DevOps (https://dev.azure.com/), you can already download 3.224.1. https://vstsagentpackage.azureedge.net/agent/3.224.1/vsts-agent-linux-x64-3.224.1.tar.gz

What is the procedure for having the **latest** tag on release?

Since Azure DevOps does not have any API for it, we rely on this one - https://api.github.com/repos/Microsoft/azure-pipelines-agent/releases/latest

**Comment by ChernenkoAV** [comment]
For pipelines-agent-linux-x64
If you need another version - change to the required one

`curl https://api.github.com/repos/Microsoft/azure-pipelines-agent/releases/latest | grep -oP 'http\S+?`**pipelines-agent-linux-x64**`\S+?\.tar\.gz' | curl $(cat) -o agent.tar.gz`
